src/NzbDrone.Core/Localization/Core/en.json

```
@@ -1842,6 +1842,14 @@
   "SeasonFinale": "Season Finale",
   "SeasonFolder": "Season Folder",
   "SeasonFolderFormat": "Season Folder Format",
+  "SeasonImportAllowAnyWarning": "Allow a season pack if it upgrades any episode. Note that this may result in stuck queue items.",
```

```suggestion
  "SeasonImportAllowAnyWarning": "Allow a season pack to be imported if it upgrades any episode. Note that this may result in stuck queue items.",
```

is the essence of this warning related to imports or grabs?
why can it result in stuck queue items, shouldn't we be working to avoid that with this setting?